frontend/src/metabase/ui/components/icons/Icon/icons/verified_collection.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="14" height="14" viewBox="0 0 14 13" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M2.2 1.75C2.06282 1.75 1.94177 1.80099 1.86103 1.87576C1.78222 1.94872 1.75 2.03478 1.75 2.11111V9.8889C1.75 9.9652 1.78222 10.0513 1.86103 10.1242C1.94177 10.199 2.06282 10.25 2.2 10.25H7V11.75H2.2C1.70066 11.75 1.21126 11.5669 0.84192 11.2249C0.47063 10.8811 0.25 10.4019 0.25 9.8889V2.11111C0.25 1.59807 0.47063 1.1189 0.84192 0.77512C1.21126 0.43313 1.70066 0.25 2.2 0.25H5.28996L7.28996 2.06818H11.8C12.2993 2.06818 12.7887 2.25132 13.1581 2.5933C13.5294 2.93708 13.75 3.41626 13.75 3.92929V6H12.25V3.92929C12.25 3.85296 12.2178 3.76691 12.139 3.69394C12.0582 3.61917 11.9372 3.56818 11.8 3.56818H6.71004L4.71004 1.75H2.2Z" fill="#F9CF48"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8 9C8 7.61929 9.11929 6.5 10.5 6.5C11.8807 6.5 13 7.61929 13 9C13 9.61856 12.7754 10.1846 12.4032 10.6211L12.869 13.1829C12.8955 13.329 12.8334 13.477 12.7105 13.5603C12.5876 13.6437 12.4272 13.6467 12.3013 13.568L10.7311 12.5867L10.5 12.5L10.2689 12.5867L8.69875 13.568C8.57284 13.6467 8.41236 13.6437 8.28949 13.5603C8.16662 13.477 8.10449 13.329 8.13105 13.1829L8.59685 10.6212C8.22466 10.1847 8 9.61858 8 9ZM10.5 7.5C9.67157 7.5 9 8.17157 9 9C9 9.82843 9.67157 10.5 10.5 10.5C11.3284 10.5 12 9.82843 12 9C12 8.17157 11.3284 7.5 10.5 7.5Z" fill="#F9CF48"/>
```

Comment:
sorry I missed this initially, but we shouldn't hard code the colors here.

There should be no `fill` on the `path` elements. the parent `svg` should have `fill="currentColor"` and we should set the color programmatically in the application code.